watchdog.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
```

Comment:
How about we put this file somewhere else... perhaps `provisioner/` I typically put all logic into folders like that... `lib/` is another option, but I fear the file would be lonely in there...Or it share it with `cache.js`.